lib/token.js

```diff
@@ -12,8 +12,37 @@ function getWellKnown(sdk) {
   return http.get(sdk, sdk.options.url + '/.well-known/openid-configuration');
 }
 
-function validateClaims(sdk, claims, aud) {
-  var iss = sdk.options.url;
+function getAuthServerUrl(sdk, authServer) {
+  authServer = authServer || sdk.options.authServer;
+  if (authServer) {
+    if (/^https?:/.test(authServer)) {
+      return authServer;
+    }
+    return sdk.options.url + '/oauth2/' + authServer;
```

Comment:
Let's add a comment here saying what this is, i.e. if they pass in a serverId instead of the full url